### PR TITLE
add /help command to message_wnd

### PR DESCRIPTION
**Pull request by TheSilentOne1**
Adds a /help command for the message window that gives information about available commands, which are currently /zoom and /pedia.

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -6184,6 +6184,11 @@ Received unexpected turn update from server.
 MESSAGES_PANEL_TITLE
 Messages
 
+MESSAGES_HELP_COMMAND
+'''/pedia [article]: open the specified article in the pedia.
+/zoom [object]: zoom to the specified universe object (system, planet, ship, fleet or building)
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Oxford comma please

**Comment by geoffthemedio** [comment]
Might be a bit confusing with the existing help command for AI debugging
..

**Comment by TheSilentOne1** [comment]
We could rename either, any suggestions for alternatives? Or the help text could appear if any invalid command was given.

**Comment by geoffthemedio** [comment]
Appears to be mostly included in / superseded by https://github.com/freeorion/freeorion/pull/2504

**Comment by Vezzra** [comment]
@TheSilentOne1, @geoffthemedio, so, can we close this as superseded...?

**Comment by TheSilentOne1** [comment]
I'll move the relevant parts to the other PR.

**Comment by TheSilentOne1** [comment]
Actually, it was completely superceded by the other PR; so no need to move anything.

**Pull request closed without merging**